hs_core/views/__init__.py

```diff
@@ -206,6 +206,7 @@ def add_files_to_resource(request, shortkey, *args, **kwargs):
         return JsonResponse(msg, status=500)
 
     except (hydroshare.utils.ResourceFileValidationException, Exception) as ex:
+        logger.exception("failed")
```

Comment:
Shouldn't this be a bit more descriptive?

Reply:
I had this in for extra debugging, didn't mean to commit it.  Removing both instances you pointed out.